README.md

```diff
@@ -90,14 +91,34 @@ region using an environment variable.
 export AWS_REGION=eu-west-1
 ```
 
+## Getting Started with aws
+
+To stand up an instance with AWS, use the `ec2_instance` type:
```

Comment:
Yeah, I knew the example needed some attention and likely had. I sent you an email on Thursday with questions about it. I can just go back to the earlier example.